server/tests/1userSignup.test.js

```diff
@@ -56,8 +56,8 @@ describe('User signup', () => {
   it('should activate the user if inactivated, and valid token supplied', (done) => {
```

Comment:
Chima! I think you must have forgotten mark suggested that the test name should not be prefixed with a number. Please, can you implement the feedback by changing the 1userSign up and other test file name you prefixed with a number to a clearer name without the number? Since we now have a user seed file. It should be fine

Reply:
@Oluwafayokemi Thanks for your feedback. I have implemented your feedback that requires changes. As for the refactoring, I figured since implementing the notification had us already touching so many files, adding that will be a bonus as it helped improve our test coverage considerably.